trolldb/config/config.py

```diff
@@ -46,18 +46,13 @@ def id_must_be_valid(id_like_string: str) -> ObjectId:
     try:
         return ObjectId(id_like_string)
     except InvalidId as e:
-        raise ValueError from e
+        raise ValueError(f"{id_like_string} is not a valid value for an ID.") from e
```

Comment:
Why change the type of error?

Reply:
I have added a note on this to the validator's docstring.

Reply:
ok, all good then.